### PR TITLE
Mailer view editing

### LINKED ISSUE #46: Email formatting request
>  When receiving M&G requests to the front desk email, it would be really nice if different sections were separated from each other. So, for example the sections could be:

>  Contact information

>  Passenger information

>  Arrival

>  Departure.

>  Having the information come in one big block makes it a little harder for me to pull information out of it when I'm looking back and forth from one screen to another.

### PR DESCRIPTION
**Pull request by Anbranin**
closes #46 

Now it looks like this in the mailer view:
screen shot 2015-11-30 at 10 31 24 am

So pretty!

### CODE REVIEW

File: app/models/field.rb [hunk 1]
```diff
@@ -43,14 +43,18 @@ def takes_placeholder?
...
 
-  def unique_prompt_name
-    "prompt_#{number}"
-  end
-
   def unique_name
     "field_#{number}"
   end
 
+  def unique_header_name
```
Comment (dfaulken) [inline app/models/field.rb]: Aren't they headings, not headers?

Comment (Anbranin) [inline app/models/field.rb]: Fair enough

File: app/views/ft_forms_mailer/send_form.text.erb [hunk 2]
```diff
@@ -3,5 +3,11 @@ Hello,
 A new Meet & Greet request has been received:
 
 <% @form_data.each do |prompt, response| %>
-  <%= prompt %>: <%= response %>
+  <% if response == :heading %>
+
```
Comment (dfaulken) [inline app/views/ft_forms_mailer/send_form.text.erb]: Don't need this newline

File: app/views/ft_forms_mailer/send_form.text.erb [hunk 3]
```diff
@@ -3,5 +3,11 @@ Hello,
...
+    <%= prompt.upcase %>
+    <%= '=' * prompt.length %>
```
Comment (dfaulken) [inline app/views/ft_forms_mailer/send_form.text.erb]: Interesting. Could we see a screenshot of how this looks?

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -24,8 +24,11 @@ def parse_form_data(data)
         responses[number.to_i] = v
       when 'prompt'
         prompts[number.to_i] = v
+      when 'header'
```
Comment (dfaulken) [inline app/helpers/application_helper.rb]: heading

File: spec/helpers/application_helper_spec.rb [hunk 5]
```diff
@@ -15,4 +15,25 @@
       expect(input_class 'time').to eql 'timepicker'
     end
   end
+
+  describe 'parse_form_data' do
+    before :each do
+      # form data sent from view to controller
+      @data = { 'prompt_1' => 'prompt value', 'field_1' => 'field value',
+                'prompt_2' => 'prompt value 2', 'field_2' => 'field value 2',
+                'header_3' => 'header value',
```
Comment (dfaulken) [inline spec/helpers/application_helper_spec.rb]: heading

File: app/views/ft_forms_mailer/send_form.text.erb (hunk 2 above)
Comment (dfaulken) [inline app/views/ft_forms_mailer/send_form.text.erb]: Never mind

File: spec/views/ft_forms_mailer/send_form_spec.rb [hunk 6]
```diff
@@ -3,7 +3,9 @@
 
 describe 'ft_forms_mailer/send_form.text.erb' do
   before :each do
-    @form_data = Hash[question1: 'answer1', email: 'my_email']
+    @form_data = [['a question', 'an answer'],
+                  ['a header', :heading],
```
Comment (dfaulken) [inline spec/views/ft_forms_mailer/send_form_spec.rb]: heading

File: spec/views/ft_forms_mailer/send_form_spec.rb [hunk 7]
```diff
@@ -11,7 +13,11 @@
   end
   it 'includes the form data of the request' do
     render
-    expect(rendered).to include 'question1: answer1',
-                                'email: my_email'
+    expect(rendered).to include 'a question: an answer'
+    'email: my_email'
+  end
+  it 'only displays the header and not the associated :heading' do
```
Comment (dfaulken) [inline spec/views/ft_forms_mailer/send_form_spec.rb]: headgin

**Comment by dfaulken** [comment]
Looks good.

File: app/views/ft_forms_mailer/send_form.text.erb (hunk 3 above)
Comment (Anbranin) [inline app/views/ft_forms_mailer/send_form.text.erb]: screen shot 2015-11-30 at 10 31 24 am

**Pull request merged**